### PR TITLE
Garbz theme updates

### PR DESCRIPTION
**Pull request by thegarbz**
A couple of tweaks to the other two themes, a new theme, and since I have a few in the folder now I added an index file.

### CODE REVIEW

File: themes/theGarbz/Focus/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Focus theme v0.1
+
+This theme is based on the classy NOX theme by NoxHirsch and has been modified to Focus on highlighting the most important information for a user. The goal was to create a theme that could easily be read from a distance. Some of the features are listed below:
```
Comment (UnchartedBull) [inline themes/theGarbz/Focus/README.md:3]: Might be nice to link the NOX theme here I think.

File: themes/theGarbz/Focus/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+* Large print time remaining indicators
+* Pre-printing screen highlighting buttons.
+* Full screen menus and adjustments with coloured backgrounds for context.
+* Red highlighting of error messages
+
+To install copy the custom-styles.css file into the octodash config folder:
+```
+~/.config/octodash/custom-styles.css
+```
+
```
Comment (UnchartedBull) [inline themes/theGarbz/Focus/README.md:16]: What Pi are you running this theme on? I noticed on the OG Raspberry Pi 3 the blur effect really takes a hit on performance. Might be worth adding as a heads-up here for other users (same would need to be added to the NOX theme then of course).

**Review by UnchartedBull — APPROVED** [review]
some minor changes. Should be good to go afterwards.

File: themes/theGarbz/Focus/README.md (hunk 1 above)
Comment (thegarbz) [reply themes/theGarbz/Focus/README.md]: Agreed and done. Commit 48ac302

File: themes/theGarbz/Focus/README.md (hunk 2 above)
Comment (thegarbz) [reply themes/theGarbz/Focus/README.md:16]: Good idea. I've noticed it wasn't quite as smooth but I didn't know why, and it's not really terrible performance just a little bit of stutter. I'm running on a Raspberry Pi 4. I added a warning commit abfc903

I also fixed a misaligned element in another commit but hopefully we should be finished now.

Comment (UnchartedBull) [reply themes/theGarbz/Focus/README.md:16]: Thanks!

**Pull request merged**